src/System.Reflection.DispatchProxy/src/Configurations.props

```diff
@@ -5,12 +5,12 @@
       netfx;
       netstandard;
       netcoreapp2.0;
-      uapaot-Windows_NT;
-      uap-Windows_NT;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations);
       netcoreapp;
+      uap-Windows_NT;
```

Comment:
This is another case where we would downgrade the impl.  In this case it would downgrade to a [platform not supported assembly](https://github.com/joperezr/corefx/blob/ba66a7d05d17453e8b49f2d9c46679b00e8916e0/src/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj#L10).  I think here we should include a specific version of the UAP build (RS3?) in the package.

Reply:
We only ever shipped UAP RS3 builds on the MP as we intentionally removed those assets from the OOB packages when we shipped NCA2.0 and in UAP we only shipped the MP, which means that perhaps for this one we have to include a live build of the package, and perhaps a placeholder on RS3 to ensure we use the one inbox.

Reply:
I see, so harvesting is not an easy option.  You could build a live copy for RS3 (like we are building a live copy for netcoreapp2.0).